freeorion.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>freeorion.desktop</id>
+  <name>FreeOrion</name>
+  <summary>Turn-based space empire and galactic conquest (4X) computer game</summary>
+  <summary xml:lang="de">Rundenbasierendes Weltraum-Strategiespiel</summary>
+  <developer_name>FreeOrion Project</developer_name>
+  <description>
+      <p>
+       FreeOrion is a free, open source, turn-based space empire and galactic conquest (4X) computer game being designed and built by the FreeOrion project. FreeOrion is inspired by the tradition of the Master of Orion games, but is not a clone or remake of that series or any other game.
+      </p>
+  </description>
+  <keywords>
+    <keyword>game</keyword>
```

Comment:
Probably wrong lower case:

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#spec-component-filespec

> A list of valid category names can be found in the Freedesktop menu spec.

https://specifications.freedesktop.org/menu-spec/latest/apa.html#main-category-registry

> Note that category names are case-sensitive.

So it's `Game`, not `game`.

Reply:
Fixed 4675bd22738d693b9c59cb3f79a8b5f768232d7f

Reply:
This isn't `⁠<categories>` though.